packages/core/src/amazonq/lsp/lspClient.ts

```diff
@@ -233,7 +233,7 @@ export async function activate(extensionContext: ExtensionContext) {
     // Create the language client and start the client.
     LspClient.instance.client = new LanguageClient(
         'amazonq',
-        localize('amazonq.server.name', 'Amazon Q Language Server'),
+        localize('amazonq.server.name', 'Amazon Q Workspace Language Server'),
```

Comment:
I renamed this to the workspace server for now. I think at some point it makes sense to consolidate it into the actual language servers?

Reply:
Well, in https://taskei.amazon.dev/tasks/IDE-15333 we actually propose to interleave LSP messages with the `Amazon Q Logs` channel.

Reply:
:+1: Will hook that up once the postreinvent branch gets merged into master